gammagl/models/infograph.py

```diff
@@ -88,20 +104,14 @@ def forward(self, x, edge_index, batch):
 
 
 class InfoGraph(nn.Module):
-    r"""
-        infograph model for unsupervised setting
+    """
+    infograph model for unsupervised setting
+
     Parameters
-    -----------
-    in_dim: int
-        Input feature size.
-    hid_dim: int
-        Hidden feature size.
-    n_layer: int
-        Number of the GNN encoder layers.
-    Functions
-    -----------
-    forward(graph):
-        graph: DGLGraph
+    ----------
+        num_feature (int): input feature dimension
+        hid_feat (int): hidden dimension
+        num_gc_layers (int): number of layers
     """
```

Comment:
The parameters in the ``rst`` doc should correspond to the parameters in ``__init__``.